### PR TITLE
Add telemetry option

### PR DESCRIPTION
**Pull request by psiinon**
Note that this does not actually add the telemetry!

@dscrobonia this also doesnt add any user information about the telemetry - I'm assuming your PR will do that :)
You can check the telemetry option via the API but right now there are no events to say when the telemetry option is changed - will you need those? Can add them in a future PR if they will be needed.

### CODE REVIEW

File: src/main/java/org/zaproxy/zap/extension/hud/HudParam.java [hunk 1]
```diff
@@ -217,6 +229,7 @@ protected void parseImpl() {
         tutorialTasks = convert(getConfig().getList(PARAM_TUTORIAL_TASKS));
         showWelcomeScreen = getConfig().getBoolean(PARAM_SHOW_WELCOME_SCREEN, true);
         enableOnDomainMsgs = getConfig().getBoolean(PARAM_ENABLE_ON_DOMAIN_MSGS, true);
+        enableTelemetry = getConfig().getBoolean(PARAM_ENABLE_TELEMETRY, true);
```
Comment (thc202) [inline src/main/java/org/zaproxy/zap/extension/hud/HudParam.java]: Disable by default if in dev mode?

**Comment by psiinon** [comment]
Closing in favour of #431 which should provide a much more comprehensive solution

**Comment by psiinon** [comment]
Arg, closed wrong one :P

**Comment by psiinon** [comment]
Finally fixed the one comment :P 
Should be ready for review.

**Comment by psiinon** [comment]
Updated again :)

File: src/main/resources/org/zaproxy/zap/extension/hud/resources/Messages.properties [hunk 2]
```diff
@@ -37,6 +37,7 @@ hud.optionspanel.button.baseDirectory	= Change
 hud.optionspanel.label.baseDirectory	= Base Directory:
 hud.optionspanel.label.enabledForDesktop = Enable when using the ZAP Desktop
 hud.optionspanel.label.enabledForDaemon = Enable when using ZAP in daemon mode
+hud.optionspanel.label.enableTelemetry = Enable anonymized telemetry
```
Comment (kingthorin) [inline src/main/resources/org/zaproxy/zap/extension/hud/resources/Messages.properties:40]: "anonymized" implies that it was identifiable and was rendered unidentified before submission, which if we take another small step implies that something could go wrong.... Would it be better to say "anonymous"? (Which to me implies that there's no identifying info to start with.)

Comment (psiinon) [reply src/main/resources/org/zaproxy/zap/extension/hud/resources/Messages.properties]: Oops, lost track of this :/
Now changed :)

**Pull request merged**